### ISSUE TITLE
Refresh on emptyDisplay

### ISSUE THREAD

**Issue by Kal-Elx** [vedartm/paginate_firestore#42]
Right now you can't refresh when emptyDisplay is shown. This is because emptyDisplay isn't a child of a [Scrollable](https://api.flutter.dev/flutter/widgets/Scrollable-class.html).

I have solved it like this but it would be nice if this was the default behaviour.

```
    RefreshIndicator(
      child: PaginateFirestore(
        ...
        emptyDisplay: LayoutBuilder(
          builder: (ctx, constraints) => SingleChildScrollView(
            physics: AlwaysScrollableScrollPhysics(),
            child: Container(
              height: constraints.maxHeight,
              child: EmptyDisplay(),
            ),
          ),
        ),
      ),
    );
```

Thanks for a great package!

**Comment by excogitatr** [comment]
I will fix this in the next version @Kal-Elx

**Comment by irshaadMax** [comment]
I also face this issue. 
On emptyDisplay it will hides my header (_SliverPersistentHeader_) too. 
Waiting owner to fix this issue.

`
PaginateFirestore(
        header: SliverPersistentHeader(
                pinned: true,
                delegate: _SliverSubMenuDelegate(/*submenu with categories*/) 
        ),

        itemBuilder: (index, context, documentSnapshot) {
          final data = documentSnapshot.data() as Map?;
          if(data == null) return Center(child: CircularProgressIndicator());
          return Text(documentSnapshot.id);
        },

        query: FirebaseFirestore.instance.collection('article')
                .where('article_category', index)
                .orderBy('article_date', descending: true),

        key:ValueKey<String>(selectedIndex.toString()),

        itemBuilderType: PaginateBuilderType.listView,

        isLive: true,

        itemsPerPage: 5,
      );
 `

**Comment by vedartm** [comment]
Hi @irshaadMax and @Kal-Elx. This is now fixed in the latest update v1.0.2.

**Comment by vedartm** [comment]
Thanks for using the package. Feel free to reopen this if you face any similar issues.
